source/_components/binary_sensor.w800rf32.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: "W800rf32 Binary Sensor"
+description: "Instructions on how to integrate W800rf32 binary sensors into Home Assistant."
+date: 2018-10-16 12:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: w800rf32.png
+ha_category: Binary Sensor
+ha_release: 0.79
```

Comment:
Pherhaps release version must be updated to 0.83, if this [PR](https://github.com/home-assistant/home-assistant/pull/17920) will soon be merged.